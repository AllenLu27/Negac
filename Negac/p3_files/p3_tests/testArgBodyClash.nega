int fn(int a, int b){
	int a;
	int b;
}
