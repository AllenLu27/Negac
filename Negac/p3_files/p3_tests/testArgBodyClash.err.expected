FATAL [2,6]: Multiply declared identifier
FATAL [3,6]: Multiply declared identifier
Semantic analysis failed
